code/src/main/webapp/index.html

```diff
@@ -4,11 +4,25 @@
 <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <link rel="stylesheet" href="./CSS/style.css">
-   <title>First JSP</title>
+   <title>Welcome</title>
 </head>
 <body>
-   <h1>Well, life goes on ... </h1>
-   <p>(0.16450910536318764)</p>
-   <a href="./JSP/test.jsp"><h3>Try Again</h3></a>
+  <div id="wrapper">
+    <div id="title">
+      <h1> Welcome </h1>
```

Comment:
This is a little bit overdone in websites - perhaps we should instead use "Contact Tracing Assistnat" or something else  that clearly idenfies the name of the site and the purpose of the site.

Think about it this way. If I wanted to tell my friend about this app, I wouldn't know what to call it right now. There's no "brand" or clear name of the project.

Reply:
done